app/installer.sh

```diff
@@ -26,6 +26,11 @@ DIR="$( cd "$( dirname "$0" )" >/dev/null 2>&1 && pwd )"
 : "${LATEST_VERSION:=https://kuma.io/latest_version}"
 : "${REPO_PREFIX:=kuma}"
 : "${CTL_NAME:=kumactl}"
+: "${BRANCH:=master}"
+
+if [ "$REPO_PREFIX" == "kong-mesh" ]; then
+  PRODUCT_NAME="KONG MESH"
+fi
```

Comment:
For kong-mesh here's how we invoke this script: 

https://github.com/Kong/docs.konghq.com/blob/main/app/mesh/installer.sh#L21-L25

So what should be done is set in docs.konghq.com an extra REPO parameter instead of doing the special casing here

Reply:
So you'd add after line 29:

```sh
: "${REPO:=kumahq/kuma}"
```

Reply:
I’ve added support for this and made the preview.sh script use installer.sh script under the hood for people that use it (muscle memory).

Do you recall another place where we might use `REPO_PREFIX` instead of `REPO` like that? I’ll take care of the Kong Mesh docs one after this.

Reply:
I'd keep PRODUCT_NAME and not have anything specific for kong-mesh in here

Reply:
I’ve added `PRODUCT_NAME` again and also added in preview.sh again.

I’ve also changed the detection of `REPO_PREFIX` to strip out anything up to and including the slash from `REPO`.

The remaining reference to Kong Mesh is the different preview version hash which we’re fixing separatetely.